modules/combined/tests/poro_mechanics/borehole_lowres.i

```diff
@@ -0,0 +1,733 @@
+# Poroelastic response of a borehole.
+#
+# LOWRES VERSION: this version does not give perfect agreement with the analytical solution
```

Comment:
Both the low res and high res versions say that they don't give perfect agreement with the analytical solution. Are one of these comments wrong? Does one of them agree with the analytical solution?